Wrapping/Generators/SwigInterface/igenerator.py

```diff
@@ -533,9 +533,17 @@ def generate_process_object_snake_case_functions(self, typedefs):
                 self.outputFile.write('    import itk\n')
                 self.outputFile.write('    import itkTemplate\n')
                 self.outputFile.write('    if isinstance(itk.%s, itkTemplate.itkTemplate):\n' % processObject)
-                self.outputFile.write('        %s.__doc__ = itk.%s.values()[0].__doc__\n' % (snakeCase, processObject))
+                self.outputFile.write('        filter_object = itk.%s.values()[0]\n' % (processObject))
                 self.outputFile.write('    else:\n')
-                self.outputFile.write('        %s.__doc__ = itk.%s.__doc__\n\n' % (snakeCase, processObject))
+                self.outputFile.write('        filter_object = itk.%s\n\n' % (processObject))
+                self.outputFile.write('    %s.__doc__ = filter_object.__doc__\n' % (snakeCase))
+                self.outputFile.write('    %s.__doc__ += "\\n Args are Input(s) to filter.\\n"\n' % (snakeCase))
+                self.outputFile.write('    %s.__doc__ += "Available Keyword Arguments:\\n"\n' % (snakeCase))
+                self.outputFile.write('    %s.__doc__ += "".join([\n' % (snakeCase))
+                self.outputFile.write('        "  " + item[3:] + "\\n"\n')
```

Comment:
Add `self.camelCaseToSnakeCase(item[3:])` to make this more Pythonic (the interface supports both CamelCase and snake_case keyword arguments).

Reply:
It's tricky: `item` only exists when the generated code runs, and the generated code doesn't have access to an IdxGenerator object because igenerator.py isn't part of the finished itk package. Should I move camelCaseToSnakeCase to itkExtras? It is already a static method.

Reply:
Alternately, I probably should generate the docstring at compile time